src/test/mod.rs

```diff
@@ -99,7 +98,8 @@ pub(crate) struct TestEnvironment {
     db: OnceCell<TestDatabase>,
     storage: OnceCell<Arc<Storage>>,
     frontend: OnceCell<TestFrontend>,
-    s3: OnceCell<TestS3>,
+    s3: OnceCell<Arc<Storage>>,
+    storage_db: OnceCell<Arc<Storage>>,
```

Comment:
`Lazy` is always better than `OnceCell`, it removes the user overhead of filling them manually

Reply:
See https://github.com/rust-lang/docs.rs/pull/911#discussion_r460587158